client/src/app/site/pages/meetings/modules/projector/modules/slides/components/list-of-speakers/modules/common-list-of-speakers-slide/components/common-list-of-speakers-slide.component.scss

```diff
@@ -28,11 +28,11 @@
 }
 
 #overlay {
-    position: absolute;
-    right: 0;
-    bottom: 100px;
+    position: fixed;
+    right: 50px;
+    bottom: 10px;
     background-color: #d3d3d3;
-    width: 40%;
+    width: 37%;
     height: 210px;
     margin: 10px;
```

Comment:
please remove the margin line to get the overlay box left-justified with motion meta table and clock.